.coveragerc

```diff
@@ -0,0 +1,20 @@
+[run]
+source = zope.testrunner
+parallel = true
+omit =
+    # These files are definitely imported and executed,
+    # but for some reason don't show up as such.
+    # See testrunner-knit
+    src/zope/testrunner/tests/testrunner-ex-pp-products/more/sampletests.py
+    src/zope/testrunner/tests/testrunner-ex-pp-products/sampletests.py
```

Comment:
Looking at `testrunner-knit.txt`, it may be that coverage.py doesn't handle the special runtime Python package's `__path__` variable manipulation magic that this test performs?

I don't quite understand what that's supposed to be used for -- pre-PEP-420 namespace-packages?

Reply:
I've speculated [about similar uses before](https://github.com/zopefoundation/ZConfig/issues/28#issuecomment-287318917) in other projects. It's not clear to me either.